lib/src/board_settings.dart

```diff
@@ -5,6 +5,15 @@ import 'models.dart';
 import 'piece_set.dart';
 import 'draw_shape_options.dart';
 
+enum InputMethod {
+  tapTwoSquares('Tap two squares'),
```

Comment:
No need for labels here, these strings should be translated on the app side.

Reply:
The labels are used in the example app though, for the choices in the selector button. This is how it's done for `BoardTheme` and `PieceSet` as well.

Reply:
You should define these labels in the example app. Board theme and piece set are different because here the labels are names that we don't translate.

Since the translation should be done outside this package we shouldn't even suggest label names. Also people might use this package with a primary language that is not English.

Reply:
Enum member should each have a doc comment explaining what they do if selected.